package.json

```diff
@@ -25,14 +25,21 @@
     "testing"
   ],
   "dependencies": {
+    "global": "^4.3.2",
+    "json-stable-stringify": "^1.0.1",
     "lodash.escaperegexp": "^4.1.2",
     "nock": "^9.0.13",
     "pkg-conf": "^2.0.0",
-    "shorthash": "^0.0.2"
+    "shorthash": "^0.0.2",
+    "typescript": "^2.3.2"
```

Comment:
@palmerj3 you missed to remove a comma after `"shorthash": "^0.0.2"`😙 `package.json` is invalid now